versioned_docs/version-v2.18.x/user-guide/install-zowe-server-install-wizard copy.md

```diff
@@ -0,0 +1,199 @@
+# Installing Zowe via Zowe Server Install Wizard
```

Comment:
@1000TurquoisePogs - Should this file AND install-zowe-server-install-wizard-1-0-0.md exist in both v2.18 and the V3 versions? (the first time I posed question, the file included incorrectly-copied content (on troubleshooting). this file now includes the correct content.)

Reply:
now, [install-zowe-server-install-wizard copy.md](https://github.com/zowe/docs-site/pull/4326/files#diff-3d570341743f74730ed97c84430848b39734b2f4bb6077b37f586c178341722c) is good, and [install-zowe-server-install-wizard-1-0-0.md](https://github.com/zowe/docs-site/pull/4326/files#diff-7d6787a30868d0ae248cc9ef9ecfa9238c93fc0bdddf66bfc3da83851033bc7b) is oudated and can be removed.